sites/geohub/src/components/pages/data/datasets/UserPermission.svelte

```diff
@@ -313,7 +313,7 @@ ${username}`;
 		try {
 			isUpadating = true;
 
-			const res = await api.add(user_email, user_permission);
+			const res = await api.add(user_email.toLowerCase(), user_permission);
```

Comment:
@iferencik I made this to force transforming to lowercase because I wanted to minimize changes. 
In hooks.server.ts, transforming email to lower case, so all email will be lower case from authentication.

If we allow users to add email mixed capitalized and lower case letter, we basically need to change many codes everywhere in GeoHub (I am not sure how many changes I need to do, but all most everywhere in GeoHub we need to change).

If we save lowercase email all ways in permission table, and all email supplied from auth is transformed to lower case, I am sure everything should work.